pkg/scheduler/framework/plugins/defaultpodtopologyspread/default_pod_topology_spread_perf_test.go

```diff
@@ -76,14 +77,13 @@ func BenchmarkTestSelectorSpreadPriority(b *testing.B) {
 				if !status.IsSuccess() {
 					b.Fatalf("unexpected error: %v", status)
 				}
-				var gotList framework.NodeScoreList
-				for _, node := range filteredNodes {
-					score, status := plugin.Score(ctx, state, pod, node.Name)
-					if !status.IsSuccess() {
-						b.Errorf("unexpected error: %v", status)
-					}
-					gotList = append(gotList, framework.NodeScore{Name: node.Name, Score: score})
+				gotList := make(framework.NodeScoreList, len(filteredNodes))
+				scoreNode := func(i int) {
+					n := filteredNodes[i]
+					score, _ := plugin.Score(context.Background(), state, pod, n.Name)
```

Comment:
Instantiating `context.Background()` everytime looks odd, cannot we reuse the outer `ctx`? Check out `wait.UntilWithContext`:

https://github.com/kubernetes/kubernetes/blob/fc8a39d439ea08b006fb7ea1db1058175cc7adf5/staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go#L98

Reply:
we can... I just missed it 😅